templates/crate/source.html

```diff
@@ -46,54 +50,54 @@
 
                         {%- for file in file_list.files -%}
                             <li class="pure-menu-item">
-                                {#
-                                    Show a link to the file with a fancy icon. If the file is a directory,
-                                    `/` is appended to show the contents of the folder
-                                #}
+                                    {#
```

Comment:
Why this new indent? (Same for comments below)

Reply:
I tried to fix the indendation in the templates with formatter, 

so indendation is consistent across the templates.

Reply:
It doesn't make much sense but doesn't matter, it's comments.